### PR TITLE
Radically streamline the PWA landing page

### LINKED ISSUE #8023: Issue with Progressive web apps (PWAs) can be improive.
Hello, MDN folks.

I'd like to report this documentation.

MDN URL: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps

#### What information was incorrect, unhelpful, or incomplete?

What does progressive web application mean. Please write thesis first then examples. I hard to understand "progressive".
Because follow sentences should place "Progressive" examples (and detail links). 

#### Specific section or headline?

first paragraph.

#### What did you expect to see?

>  Please write thesis first then examples. I hard to understand "progressive".
>  Because follow sentences should place "Progressive" examples (and detail links). 

#### Did you test this? If so, how?

No. I didn't mention sample programs.

<details>
<summary>MDN Content page report details</summary>

* Folder: `ja/web/progressive_web_apps`
* MDN URL: https://developer.mozilla.org/ja/docs/Web/Progressive_web_apps
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/ja/web/progressive_web_apps/index.html
* Last commit: https://github.com/mdn/translated-content/commit/01aad4e9d467a4b4a5b237f888b6c8fda3b07815
* Document last modified: 2021-08-07T20:22:47.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps is a landing/portal page that should just serve as a gateway to attract users into the reading the subpages where all the useful content actually is.

But that existing page has so much content on it — and so much clutter — that it’s not serving its purpose well at all.

So this change pares that down to just a well-structured, slim list of links for quickly getting readers over to the where the meat of the content is.

Among the changes:

- Drop huge PWA logo taking up 25% of screen real estate above the fold
- Copy-edit/rewrite intro paragraphs down to just two sentences
- Drop link redundancy/wordiness caused by {{LandingPageListSubpages}} macro
- Assemble links to subpages into a better order/structure for readers
- Discard cruft from list of links to external sites
- Distill external-sites list down to a curated set of just the best stuff

Fixes https://github.com/mdn/content/issues/8023

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/index.html [hunk 1]
```diff
@@ -10,85 +10,46 @@
...
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Responsive/responsive_design_building_blocks">Understanding the building blocks of responsive design</a></li>
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Offline_Service_workers">How to make PWAs work offline (using service workers)</a></li>
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Installable_PWAs">How to make PWAs installable</a>, <a href="/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen">enable “add to home screen”</a>, and <a href="/en-US/docs/Web/Progressive_web_apps/Installing">further details on installing PWAs</a></li>
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Re-engageable_Notifications_Push">How to make PWAs “re-engageable” (using the Notifications API and Push API)</a></li>
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Loading">How to enable progressive loading</a></li>
+ <li><a href="/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first">How to build “mobile first” PWAs</a></li>
+</ul>
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/index.html:30]: Why separate this into two lists? If there are distinct, they probably need a kind of heading.

File: files/en-us/web/progressive_web_apps/index.html [hunk 2]
```diff
@@ -10,85 +10,46 @@
...
- <li><a href="https://marketplace.visualstudio.com/items?itemName=mayeedwin.vscode-pwa">PWA VS Code extension</a> - A collection of all essential code snippets you need to build Progressive Web Apps right there in your VS Code environment.</li>
+ <li><a href="https://web.dev/progressive-web-apps/">Progressive Web Apps landing page</a> and <a href="https://web.dev/pwa-checklist/">check list</a> on <a href="https://web.dev/">web.dev</a></li>
+ <li><a href="https://www.csschopper.com/blog/progressive-web-apps-everything-you-need-to-know/">Progressive Web Apps: Everything You Need To Know</a></li>
+ <li><a href="https://medium.com/@slightlylate/progressive-apps-escaping-tabs-without-losing-our-soul-3b93a8561955#.6czgj0myh">Progressive Web Apps: Escaping Tabs Without Losing Our Soul</a></li>
+ <li><a href="https://developers.google.com/web/updates/2015/11/app-shell">Instant Loading Web Apps with An Application Shell Architecture</a></li>
+ <li><a href="https://developers.google.com/web/ilt/pwa/introduction-to-progressive-web-app-architectures">Introduction to Progressive Web App Architectures</a></li>
+ <li><a href="https://serviceworke.rs/">Service worker cookbook</a></li>
 </ul>
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/index.html:47]: Ditto

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just the separation in two times two lists…

Btw, I am happy the giant logo is going away.

File: files/en-us/web/progressive_web_apps/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/progressive_web_apps/index.html]: > Why separate this into two lists?

Thanks for catching that (both those) — I think was just overlooked extra cruft left over from how the links had previously been marked up. Fixed now

File: files/en-us/web/progressive_web_apps/index.html (hunk 2 above)
Comment (sideshowbarker) [reply files/en-us/web/progressive_web_apps/index.html]: > Ditto

Yup — fixed here too

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
I like these changes.

**Comment by sideshowbarker** [comment]
> Btw, I am happy the giant logo is going away.

Yeah that logo was just kind of obnoxiously large. I considered trying to fit a much-smaller version of it into the page, but I think I lack the design sense to pull it off. Maybe now that we have page trimmed down to something with better readability, somebody with design chop can come back in later and get some form of that logo restored into it.

> I like these changes.

Yeah personally I had avoided even trying to ever use this article — because it just made me shudder. But in looking into to it, I seemed to find there was some real gold there that was just waiting to be brought out into the sunshine. 😆

Anyway, thanks much for the review — and thanks much in general for the time and conscientiousness you you put into your reviews. For me, you bring a lot of positive energy to the project, and a feeling for me of always genuinely looking forward to the next interaction with you 😀

**Pull request merged**